src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const whoIsPartner = sex === 'f'
+    ? `${partnerName} is my husband`
+    : `${partnerName} is my wife`;
+
+  return (
+    <>
+      <section className="Person">
```

Comment:
```suggestion
    <section className="Person">
```
There is no need to use React fragment here as you already have a container